src/main/java/racingcar/WinnerExtraction.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class WinnerExtraction {
+    public int getMaxPosition(List<Car> carList) {
+        int maxPosition = 0;
+
+        for (Car car : carList) {
+            if (maxPosition < car.getPosition()) {
+                maxPosition = car.getPosition();
+            }
+        }
+        return maxPosition;
+    }
+
+    public List<String> getWinners(List<Car> carList, int maxPosition) {
+        List<String> winners = new ArrayList<>();
+        for (Car car : carList) {
+            if (car.getPosition() == maxPosition) {
+                winners.add(car.getCarName());
+            }
+        }
+        return winners;
+    }
```

Comment:
```suggestion
    private int getMaxPosition(List<Car> carList) {
        int maxPosition = 0;

        for (Car car : carList) {
            if (maxPosition < car.getPosition()) {
                maxPosition = car.getPosition();
            }
        }
        return maxPosition;
    }

    public List<String> getWinners(List<Car> carList) {
        int maxPosition = getMaxPosition(carList);
        List<String> winners = new ArrayList<>();
        for (Car car : carList) {
            if (car.getPosition() == maxPosition) {
                winners.add(car.getCarName());
            }
        }
        return winners;
    }
```
`getMaxPosition`을 외부에 노출 시키지 않고, `WinnerExtraction`객채의 책임인 "우승자동차들을 전달한다."만 잘 수행하면 될 것 같습니다.